api/system_can.dbc

```diff
@@ -0,0 +1,317 @@
+VERSION ""
+
+
```

Comment:
I would probably put this in the git ignore at some point, we could keep this for now but the user should import the dbc file.